src/clientSideScene/sceneInfra.ts

```diff
@@ -230,7 +231,10 @@ export class SceneInfra {
       // Project that position to screen space
       vector.project(this.camControls.camera)
 
-      const _angle = typeof angle === 'number' ? angle : getAngle(from, to)
+      let _angle = 45
```

Comment:
this makes startProfile overlays offset at 45 degrees, it different from the other overlays because they are usual based on the angle of the segment, but there's no segment here.